libexec/scoop-info.ps1

```diff
@@ -25,23 +26,27 @@ if ($app -match '^(ht|f)tps?://|\\\\') {
     $global = installed $app $true
     $app, $bucket, $null = parse_app $app
     $status = app_status $app $global
-    $manifest, $bucket = find_manifest $app $bucket
+    $app, $manifest, $bucket, $url = Find-Manifest $app $bucket
```

Comment:
If Find-Manifest can check all - UNC paths, remote URLs, normal filepath, and app name, I can just remove the if-else. Would that be good?

Reply:
Since `Find-Manifest`'s faulty (`parse_app` should be added into function body, IMO), you must preprocess `app` with `parse_app` and this needs check UNC first 😭 

So maybe modify `Find-Manifest` and remove all preprocessing `parse_app` before function references in other files and drop if-else here?

Reply:
> So maybe modify `Find-Manifest` and remove all preprocessing `parse_app` before function references in other files and drop if-else here?

That sounds like a change big enough for a separate PR, what do you think?

Reply:
Okay, I'm thinking of a separate PR first, but I'd like to ask if there are some other tricks. If not, leave it here and make a new PR to refactor `Find-Manifest` (may replace `find_manifest` in some other places.)

Reply:
I could not think of any tricks. New PR would be best idea.